README.md

```diff
@@ -41,10 +41,43 @@ If you specify a resolution less than 640 x 480, it will default to 320 x 240.
 
 Frame rates greater than noted above will default to the next slowest valid frame rate.
 
+The number of cameras that can be used simultaneously at each framerate and resolution is limited by the 480Mbit/s data transfer rate of USB 2.0. To calculate possible values you can use the equation: width * height * FPS * # of cameras * 8 / 1,000,000. If the resulting value is less than 480 it is theoretically possible. In practice, it's been observed that values shouldn't exceed 80% of the 480Mbit/s limit to account for bandwidth and processing power.
```

Comment:
In practice, it's been observed that the data rate usually maxes out around 384Mbit/s (about 80% of the theoretical maximum) which may be due to other items using the USB bus (pointing devices, keyboards, disks, etc), processing power or other factors.